confignode/src/assembly/resources/conf/iotdb-confignode.properties

```diff
@@ -17,6 +17,14 @@
 # under the License.
 #
 
+####################
+### Cluster Configuration
+####################
+
+# Used for indicate cluster name and distinguish different cluster.
+# Datatype: string
+cluster_name=defaultCluster
```

Comment:
All the keys in iotdb-confignode.properties have `cn_` prefix. It's better to change `cluster_name` to `cn_cluster_name`, or just move it to iotdb-commom.properties ?

Reply:
I've moved it into the iotdb-common.properties~